packages/wallet/src/services/TransactionReemitter.ts

```diff
@@ -24,7 +24,7 @@ interface TransactionReemitterProps {
   submitting$: Observable<Cardano.NewTxAlonzo>;
   store: DocumentStore<NewTxAlonzoWithSlot[]>;
   tipSlot$: Observable<Cardano.Slot>;
-  stabilityWindowSlotsCount?: number;
```

Comment:
Please mark the commit as breaking change.

Reply:
Yep, will reword the commit name once the PR is ready for merge, to avoid force pushing during the review process.